README.md

```diff
@@ -35,6 +58,36 @@ This shell script contains commands that build and runs the fala app.
 
 If you're not using `Docker Desktop`, you may want to have two terminals open.
 
+### PlayWright Testing with Docker
+
+Playwright has its own Docker container. Tests are not executed in the FALA Docker container but instead inside a 
+Playwright Docker container, which looks for FALA's URLs to test against.
+
+You will need the FALA Docker container running in order for the tests to work, as the Playwright Docker container 
+interacts with FALA's URLs.
+
```

Comment:
I think it's worth adding that you need Docker Desktop up and running before running this command `docker-compose up -d --build `

Reply:
But I do say that `You will need the FALA Docker container running in order for the tests to work, as the Playwright Docker container interacts with FALA's URLs.`

But I've appended the following command above to really help make it clearer. Thanks.